pyproject.toml

```
@@ -1,6 +1,6 @@
 [project]
 name = "spikeinterface"
-version = "0.101.0"
+version = "0.101.0.dev0"
```

Can you revert this? See other PR 
```suggestion
version = "0.101.0"
```